packages/receipts/package.json

```
@@ -1,11 +1,11 @@
 {
   "name": "@graphprotocol/receipts",
   "version": "0.0.1",
-  "description": "Graph Network State Channel Receipt Manager",
+  "description": "State Channel Receipt Manager",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
-  "repository": "https://github.com/statechannels/graph-payments",
-  "author": "State Channels",
+  "repository": "https://github.com/graphprotocol/indexer",
+  "author": "Graph Protocol",
```

Unintentional change
```suggestion
  "repository": "https://github.com/statechannels/graph-payments",
  "author": "State Channels",
```